README.md

```diff
@@ -51,7 +51,7 @@ class SomeClass {
 }
 ```
 
-* **1.6** When writing a type for a property, a key for a dictionary, a function argument, a protocol conformance, or a superclass, don't add a space before the colon.
+* **1.6** When writing a type for a property, contant or variable, a key for a dictionary, a function argument, a protocol conformance, or a superclass, don't add a space before the colon.
```

Comment:
1. spelling: contant -> constant
2. I'd just write it as "... property, constant, variable, a key ..."